src/Native/Windows/build-native.cmd

```diff
@@ -111,6 +111,13 @@ echo See: https://github.com/dotnet/coreclr/blob/master/Documentation/project-do
 exit /b 1
 
 :GenVSSolution
+:: Generate Version file
+if not exist "%__IntermediatesDir%\clrcompression" md "%__IntermediatesDir%\clrcompression"
+msbuild /t:GenerateVersionHeader build.proj /p:NativeVersionHeaderFile="%__IntermediatesDir%\clrcompression\_version.h" /p:GenerateVersionHeader=true
+:: Copy NativeVersion.rc to the Intermediate dir
+if [%TOOLS_DIR%]==[] set TOOLS_DIR=%__binDir%\..\Tools\
+if not exist "%__IntermediatesDir%\clrcompression\NativeVersion.rc" copy "%TOOLS_DIR%NativeVersion.rc" "%__IntermediatesDir%\clrcompression\NativeVersion.rc"
```

Comment:
Makes sense. With my latest commit I removed this entirely and now I'm referencing the one from Tools directly, so if we add more native projects they will all reference the same one and there are no multiple copies.